src/lib/controllers/environment-file/environment.ts

```diff
@@ -0,0 +1,30 @@
+import {
+	DockerDeployTarget,
+	EnvConfigStore,
+	EnvironmentObject,
+	KubernetesConfigStore,
+	KubernetesDeployTarget,
+} from '.';
+
+export class Environment implements EnvironmentObject {
+	name: string;
```

Comment:
This and all the following are `public`. Is this correct?

Reply:
5) Removed!